CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.3 (2016-03-15)
```

Comment:
I'd change this to `## current` just in case - I may have to do a full release if some big change comes in :)